.dev/sass/components/_menus.scss

```diff
@@ -315,6 +315,60 @@ body.no-max-width .main-navigation {
 	}
 }
 
+.navigation.pagination {
+	clear: both;
+	margin: 0 0 1.5em;
+
+	.nav-links {
+		text-align: center;
+
+		.page-numbers {
+			display: inline-block;
+			border: none;
+			border-radius: 3px;
+			background-color: $color__background-body;
```

Comment:
@EvanHerman The only reason we used a background color for the Primer styles is because it has two background colors, one for the page and another for the content. Activation does not have a `content_background_color` which [it unsets](https://github.com/godaddy/wp-activation-theme/blob/12f3aaf2478bf5c62c0def3c6fe94a23f466c0d2/functions.php#L117) from the parent scheme. This line can be removed and the background can just be inherited.

 ![](https://dl.dropboxusercontent.com/s/xu12o8crydsvkpm/Screen%20Shot%202017-03-01%20at%2010.28.05%20AM.png?dl=0)